manual/parts/threads.tex

```diff
@@ -169,15 +169,15 @@ \subsection{Scheduling Contexts}
 
 Threads that have short execution times (e.g interrupt handlers) and are not frequently preempted
 should have less refills, while longer running threads with long values of $b$ should have a higher
-value. Threads bound to a scheduling context with 0 extra refills will behave periodically -- tasks
+value. Threads bound to a scheduling context with 0 extra refills will get run periodically -- tasks
 that use their head replenishment, or call yield, will not be scheduled again until the start of
 their next period.
 
 Given the number of replenishments is limited, if a node's SC changes and the outgoing SC does not
 have enough space to store the new replenishment, space is created by removing the current
 replenishment which can result in preemption if the next replenishment is not yet available.
 Scheduling contexts with a higher number of configured refills will consume closer
-to their whole budget, as they can be preempted or switch threads more often without filling their
+to their whole budget, as they can be preempted or else? switch threads more often without filling their
```

Comment:
as a logician I'm tempted to leave the original `or`, but I think language pragmatics probably want `and` here:

```suggestion
to their whole budget, as they can be preempted and switch threads more often without filling their
```